jest.config.js

```diff
@@ -0,0 +1,180 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
```

Comment:
I personally prefer Jest config to be inside `package.json`, and also all these comments are not needed :)

Reply:
This may be because of the new version of jest. Test not work well when I install jest using the config in the package.json.

Reply:
I'll try later.